health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FileComponent.js

```
@@ -1,89 +1,130 @@
-            <Card type={"secondary"} className="card-color">
-                <div className="dm-parent-container">
-                    {/* Left side: File Icon */}
-                    <div
-                        className="dm-uploaded-file-container-sub"
-                        onClick={() => setShowPreview(true)}
-                    >
-                        <FileIcon className="dm-icon" />
-                        <div>{fileName}</div>
-                    </div>
+    const onFileDownload = (file) => {
+        if (file && file?.url) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)